src/Chapter2.hs

```diff
@@ -783,9 +803,15 @@ divideTenBy :: Int -> Int
 divideTenBy x = div 10 x
```

Comment:
I see that task on eta-reduction is not finished and you have some questions about this topic. I will try to explain it a bit more using this function as an example.

When eta-reducing, you need to remove arguments from both sides of the `=` sign. So the correct solution here is:

```haskell
divideTenBy = div 10
```

See how it compares with the original code:

```diff
-divideTenBy x = div 10 x
+divideTenBy = div 10
```

The motivation behind eta-reduction if that if you have two functions `foo` and `bar` of the same type, and they return the same results for any arguments (so `foo x` always equal to `bar x` for any `x`). In such situations, we say that functions `foo` and `bar` are equal, and if we want to define `foo` in terms of `bar`, we can write `foo = bar` instead of a longer `foo x = bar x` (which means that `foo` returns the same result as `bar` for any `x`).

Does it make sense?